les/server_handler.go

```diff
@@ -128,6 +128,7 @@ func (h *serverHandler) handle(p *peer) error {
 	}
 	// Reject light clients if server is not synced.
 	if !h.synced() {
+		p.Log().Debug("Not synced Light Ethereum rejecting", p)
```

Comment:
somehow this sentence does not feel like proper english
what about "rejecting light client that is not synced"

Reply:
It's the server that's not synced. I agree it doesn't sound proper English, though.

Reply:
How about "Light server not synced, so it's rejecting client ..."?

Reply:
"Light server not synced, rejecting client" sounds fine to me